plugin/pkg/scheduler/factory/factory.go

```diff
@@ -240,6 +348,7 @@ func (c *ConfigFactory) addPodToCache(obj interface{}) {
 	if err := c.schedulerCache.AddPod(pod); err != nil {
 		glog.Errorf("scheduler cache AddPod failed: %v", err)
 	}
+	// NOTE: Updating equivalence cache of addPodToCache has been handled optimistically in InvalidateCachedPredicateItemForPodAdd.
```

Comment:
If a pod can bypasse scheduler, then scheduling algorithms (including eclass) shouldn't have affected this pod e.g. static pod, am I right here?